07-algorithms/algorithms-checkpoint.md

```diff
@@ -2,8 +2,7 @@
 
 | Logistics             | Details                                         |
 | --------------------- | ----------------------------------------        |
-| Project details       | https://github.com/Ada-C16/dynamic-programming  |
-| JS Version            | https://github.com/Ada-C16/dynamic-programming-js  |
+| Project details       | https://github.com/ada-c17/Newman-Conway-Sequence  |
```

Comment:
Consider making a link for student convenience.

Reply:
Done, thank you for the callout!